### PR TITLE
Feat/api#2

### PR DESCRIPTION
**Pull request by chlyNiklas**
Proccessor's api implementation

### CODE REVIEW

File: processor/api.py [hunk 1]
```diff
@@ -1,46 +1,63 @@
...
                 return
 
-            self.respond(get(0))
+            try:
+                offset = int(parse_qs(url.query)["offset"][0])
+            except Exception:
+                pass
+
+            print(url.params)
+            print(offset)
```
Comment (Jakoshi45) [inline processor/api.py]: these will be removed later right?

File: processor/api.py [hunk 2]
```diff
@@ -1,46 +1,63 @@
...
 
-                data = json.loads(post_data.decode("utf-8"))
+                flat = json.loads(post_data.decode("utf-8"))
+                data = Capture.from_dict(flat)
+                detector.calculate(data)
+
+                self.respond("ok".encode("utf-8"), 200)
+
             except Exception as e:
                 print(f"An error occurred: {e}")
```
Comment (Jakoshi45) [inline processor/api.py]: is the f before the quotation marks intentional?

File: processor/api.py [hunk 3]
```diff
@@ -1,46 +1,63 @@
...
+                print(e)
```
Comment (Jakoshi45) [inline processor/api.py]: isn't this redundant?

**Review by Jakoshi45 — CHANGES_REQUESTED** [review]

File: processor/api.py (hunk 2 above)
Comment (chlyNiklas) [reply processor/api.py]: Jes, this is for python formated strings

File: processor/api.py (hunk 3 above)
Comment (chlyNiklas) [reply processor/api.py]: fixed

File: processor/api.py (hunk 1 above)
Comment (chlyNiklas) [reply processor/api.py]: fixed

**Review by Jakoshi45 — APPROVED** [review]

**Pull request merged**